e2e/betweenness_centrality_online_test/test_online_get/input.cyp

```diff
@@ -7,6 +7,7 @@ queries:
         MERGE (a: Node {id: 3}) MERGE (b: Node {id: 4}) CREATE (a)-[:RELATION]->(b);
         MERGE (a: Node {id: 3}) MERGE (b: Node {id: 5}) CREATE (a)-[:RELATION]->(b);
         MERGE (a: Node {id: 4}) MERGE (b: Node {id: 5}) CREATE (a)-[:RELATION]->(b);
+        CALL betweenness_centrality_online.set() YIELD *;
```

Comment:
What happens if `set` is not called here

Reply:
When `get` is called without there being previously computed BC scores, it defaults to computing them with `set` using the default parameters.
See `test_get_uninitialized` for an example.

Reply:
What happens if update is called on uninitialized graph?

Reply:
Same thing ([test_online_update_uninitialized](https://github.com/memgraph/mage/tree/T261-MAGE_dynamic-BC-bug/e2e/betweenness_centrality_online_test/test_online_update_uninitialized)).